vaadinfrontend/frontend/themes/datamanager/components/card.css

```diff
@@ -9,6 +9,7 @@
   overflow: hidden;
   text-overflow: ellipsis;
   margin: 4px 4px 4px 4px;
+  cursor: pointer;
```

Comment:
Why a pointer? It is not guaranteed that the user can interact with a card, thus I would not make it as default for the complete card.